src/.nuget/packages.config

```diff
@@ -2,6 +2,8 @@
 <packages>
   <package id="Microsoft.DotNet.BuildTools" version="1.0.22-prerelease" />
   <package id="Microsoft.DotNet.TestHost" version="1.0.2-prerelease" />
+  <package id="OpenCover" version="4.5.3522" />
+  <package id="ReportGenerator" version="2.0.4.0" />
```

Comment:
It is a shame we don't have conditionals in the packages.config. Now everyone needs to download these even if they don't do code coverage builds.